main.py

```diff
@@ -36,48 +37,66 @@
     '--debug', dest='debug', default=False,
     help='Debugging log level'
 )
+parser.add_argument(
+    '--plot', dest='plot',
+    default='', help='Plot to send to ChatGPT to create scenes',
+)
+parser.add_argument(
+    '--script-out', dest='scriptout',
+    default='', help='Path to the YAML script to write',
+)
 args = parser.parse_args()
 
-if args.script == '':
-    print("Missing script file")
-    sys.exit(1)
 
-valid_text_options = ['chatgpt', 'fake']
-if args.textai not in valid_text_options:
-    print("Please provide a valid option:" +
-          "chatgpt or fake")
-    sys.exit(1)
+def main():
+    if args.plot != '':
+        if args.scriptout == '':
+            print("Missing output script file")
+            sys.exit(1)
+        gpt = ScriptChatGPT()
+        with open(args.out, 'w') as file:
+            file.write(gpt.generate(args.plot) + '\n')
+        logging.info("Done")
```

Comment:
This works, but instead of `else` just quit here since there is nothing else to do: `sys.Exit(0)`

Using `(0)` means that there was no error in the program. Exiting with a value other than zero means to the operating system that the program failed.